hazelcast/src/main/java/com/hazelcast/jet/core/ProcessorMetaSupplier.java

```diff
@@ -288,16 +348,16 @@ static ProcessorMetaSupplier forceTotalParallelismOne(@Nonnull ProcessorSupplier
      * @param supplier the supplier that will be wrapped
      * @param partitionKey the supplier will only be created on the node that owns the supplied
      *                     partition key
+     * @param permission the required permission to run the processor
      * @return the wrapped {@code ProcessorMetaSupplier}
      *
      * @throws IllegalArgumentException if vertex has local parallelism setting of greater than 1
      */
     @Nonnull
     static ProcessorMetaSupplier forceTotalParallelismOne(
-            @Nonnull ProcessorSupplier supplier, @Nonnull String partitionKey
```

Comment:
I think we should add forceTotalParallelismOne(ProcessorSupplier supplier, String partitionKey) back to not break anything.